ngraph/core/include/ngraph/op/interpolate.hpp

```diff
@@ -83,22 +85,27 @@ namespace ngraph
             private:
                 InterpolateAttrs m_attrs;
             };
-        }
+        } // namespace v0
 
-        namespace v3
+        namespace v4
         {
             class NGRAPH_API Interpolate : public Op
             {
             public:
-                static constexpr NodeTypeInfo type_info{"Interpolate", 3};
+                static constexpr NodeTypeInfo type_info{"Interpolate", 4};
```

Comment:
Please use `NGRAPH_RTTI_DECLARATION` and  `NGRAPH_RTTI_DEFINITION`
In hpp file: `NGRAPH_RTTI_DECLARATION;`
In cpp file: `NGRAPH_RTTI_DEFINITION(op::v4::Interpolate, "Interpolate", 4);`

Reply:
Done.